docs/configuration.rst

```diff
@@ -15,166 +15,250 @@ before configuring django-csp.
    policies and even errors when mistakenly configuring them as a ``string``.
 
 
+Migrating from django-csp <= 3.8
+================================
+
+Version 4.0 of django-csp introduces a new configuration format that breaks compatibility with
+previous versions.  If you are migrating from django-csp 3.8 or lower, you will need to update your
+settings to the new format. See the :ref:`migration guide <migration-guide-chapter>` for more
+information.
+
+Configuration
+=============
+
+All configuration of django-csp is done in your Django settings file with the
+``CONTENT_SECURITY_POLICY`` setting or the ``CONTENT_SECURITY_POLICY_REPORT_ONLY`` setting. Each of these
+settings expects a dictionary representing a policy.
+
+The ``CONTENT_SECURITY_POLICY`` setting is your enforcable policy.
+
+The ``CONTENT_SECURITY_POLICY_REPORT_ONLY`` setting is your report-only policy. This policy is
+used to test the policy without breaking the site. It is useful when setting this policy to be
+slightly more strict than the default policy to see what would be blocked if the policy was enforced.
+
+The following is an example of a policy configuration with a default policy and a report-only
+policy. The default policy is considered a "relaxed" policy that allows for the most flexibility
+while still providing a good level of security. The report-only policy is considered a step towards
+a more slightly strict policy and is used to test the policy without breaking the site.
+
+.. code-block:: python
+
+    CONTENT_SECURITY_POLICY = {
+        "EXCLUDE_URL_PREFIXES": ["/admin/"],
```

Comment:
Suggestion, non-blocking: I wonder if using `/admin/` as an example here hints to users that excluding admin is fine and dandy to do, when  it might not be so, depending on what apps they are using under `/admin/*`

Maybe tweaking that to /some-path/ here (and in the RO config below) is worth it?

Reply:
Good suggestion thanks!